src/aws/secrets_helpers.go

```diff
@@ -15,14 +15,9 @@ import (
 
 const secretsFile string = "secrets.json"
 
-// CreateSecretsStore creates an S3 bucket  and file object used for secrets management
-func CreateSecretsStore(awsConfig types.AwsConfig) error {
-	s3client := createS3client(awsConfig)
-	return createS3Object(s3client, strings.NewReader("{}"), awsConfig.SecretsBucket, secretsFile)
-}
-
-// ReadSecrets reads secret key value pair from remote store
-func ReadSecrets(awsConfig types.AwsConfig) (types.Secrets, error) {
+// GetOrCreateSecrets reads secret key value pair from remote store
+// It creates an empty secrets store if necessary
+func GetOrCreateSecrets(awsConfig types.AwsConfig) (types.Secrets, error) {
```

Comment:
Can we just keep this as `ReadSecrets`, and either extract a `CreateSecretsStore` function or just make it very clear in the comments that `ReadSecrets` creates the store if it doesn't exist? I think it's actually more confusing to call this `GetOrCreateSecrets`. It kind of sounds like this function could be used for POST-ing a secret as well as GETting one